consumer/pdata/common.go

```diff
@@ -604,19 +604,19 @@ func (am AttributeMap) Sort() AttributeMap {
 // Len returns the length of this map.
 //
 // Because the AttributeMap is represented internally by a slice of pointers, and the data are comping from the wire,
-// it is possible that when iterating using "ForEach" to get access to fewer elements because nil elements are skipped.
+// it is possible that when iterating using "Range" to get access to fewer elements because nil elements are skipped.
 func (am AttributeMap) Len() int {
 	return len(*am.orig)
 }
 
-// ForEach iterates over the every elements in the map by calling the provided func.
+// Range iterates over the every elements in the map by calling the provided func.
 //
 // Example:
 //
-// it := sm.ForEach(func(k string, v AttributeValue) {
+// it := sm.Range(func(k string, v AttributeValue) {
 //   ...
 // })
-func (am AttributeMap) ForEach(f func(k string, v AttributeValue)) {
+func (am AttributeMap) Range(f func(k string, v AttributeValue)) {
```

Comment:
If we aim for consistency also make `f` return bool to stop iteration like sync.Map.Range does. That's useful for search operations.

Reply:
@tigrannajaryan sure, changed :)